projects/media-viewer/src/lib/styles/media.scss

```diff
@@ -350,3 +358,15 @@
     display: none;
   }
 }
+
+@media all and (max-width: 1000px) {
+  div.hiddenIfLargerThan1000px, div.hiddenIfLargerThan1000px * {
```

Comment:
just to confirm ive tested it and it works with class names. sorry about my previous comment.